teams/tidb/membership.json

```diff
@@ -116,6 +116,7 @@
         "longfangsong",
         "mengxin9014",
         "mornyx",
+        "CabinfeverB",
```

Comment:
Please follow the the alphabetical(and case) order, move this betwen "BornChanger" and "ChenPeng2013"